third-party/cmake/LibvtermCMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(libvterm LANGUAGES C)
```

Comment:
should this file be upstreamed? Or, why isn't it a part of `BuildLibvterm.cmake` ?